projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.7-beta - 2022-02-22
+### Enhancements
+- Dashboard: improve loading of plugins page.
+- Dashboard: remove marketplace related add plugins menu redirect.
```

Comment:
Since that's WoA-specific, I would move it to other.

Reply:
Thanks, didn't know quite what this was.